js/main.js

```diff
@@ -30,16 +29,19 @@ function deleteBook(mybook) {
   mybook.target.offsetParent.parentElement.remove();
 }
 
+function ConfirmDelete() {
+  alert('Record will be deleted!');
+}
+
```

Comment:
- [ ] Don't use alert() this is because it stops code execution and displays differently on different browsers. Use custom modal instead.
![alert_requirement](https://user-images.githubusercontent.com/32361963/105152097-d3293280-5b17-11eb-9570-ff4406796d6d.png)